sdk/communication/azure-communication-chat/src/samples/java/com/azure/communication/chat/ReadmeSamples.java

```diff
@@ -219,8 +219,8 @@ public void listChatThreadMember() {
     public void addChatThreadMembers() {
         ChatThreadClient chatThreadClient = getChatThreadClient();
 
-        CommunicationUser user1 = new CommunicationUser("Id 1");
-        CommunicationUser user2 = new CommunicationUser("Id 2");
+        CommunicationUserIdentifier user1 = new CommunicationUserIdentifier("Id 1");
+        CommunicationUserIdentifier user2 = new CommunicationUserIdentifier("Id 2");
 
```

Comment:
Which README.md has reference to this code? Anything in that README.md should be changed accordingly?

Reply:
No, these lines are not used directly in the README just there for complication purposes.